### ISSUE TITLE
Mitigation score needs to be deducted based on rule

### ISSUE THREAD

**Issue by gcharest** [canada-ca/aia-eia-js#15]
There are two mains sections: all pages before the De-risking/mitigation pages add up to the Raw Risk

The last two pages score adds to the mitigation score.

The Total score rule is:

```python
percentage = 0.8
deduction = 10
maximumMitigation = questions.map(max(value))   /*currently 28*/
if (mitigationScore >= percentage * maximumMitigation):
    totalScore = rawScore - deduction
else:
    totalScore = rawScore
```

**Comment by gcharest** [comment]
Current `percentage` and `deduction` values should be editable in the future.

**Comment by gcharest** [comment]
Should look at fix #18 to identify "raw risk score" vs "mitigation score"

Total Risk score can then be calculated based on algorithm above.

Finally, level of risk (1 up to 4) is defined by ranges:
1. 0-18
2. 19-36
3. 37-54
4. 55 - max
